notebooks-auth/test_thredds.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Copy/Paste error I think, should be "readable" not "browsable" again at this point.  Browsable was tested in previous cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>You're right, will change title to use 'readable' </p><p> </p>